src/board/system76/lemp11/board.mk

```diff
@@ -28,5 +28,19 @@ CFLAGS+=\
 	-DCHARGER_CHARGE_VOLTAGE=8800 \
 	-DCHARGER_INPUT_CURRENT=0x300
 
+# Fan curve
+CFLAGS+=-DBOARD_HEATUP=5
+CFLAGS+=-DBOARD_COOLDOWN=20
+CFLAGS+=-DBOARD_FAN_POINTS="\
+	FAN_POINT(45, 25), \
+	FAN_POINT(50, 30), \
+	FAN_POINT(55, 30), \
+	FAN_POINT(60, 40), \
+	FAN_POINT(65, 50), \
+	FAN_POINT(70, 60), \
+	FAN_POINT(75, 70), \
+	FAN_POINT(80, 80) \
```

Comment:
These all increase duty by 10% for every 5°, so I think you only need the last one with  interpolation enabled.

Reply:
I've been testing this, and I get very similar temperature results with the following.

```
# Custom fan curve
CFLAGS+=-DBOARD_HEATUP=5
CFLAGS+=-DBOARD_COOLDOWN=20
CFLAGS+=-DBOARD_FAN_POINTS="\
	FAN_POINT(45, 25), \
	FAN_POINT(80, 80) \
"
```
I've been reading old PRs regarding interpolation, https://github.com/system76/ec/pull/190 for example, but I think I'm missing something. How do I enable interpolation?

When I try with just the last one I don't see nearly the same improvements in terms of temperature.

```
CFLAGS+=-DBOARD_HEATUP=5
CFLAGS+=-DBOARD_COOLDOWN=20
CFLAGS+=-DBOARD_FAN_POINTS="\
	FAN_POINT(80, 80) \
"
```

Reply:
Interpolation is enabled by default (`SMOOTH_FANS` is set to 1).

So

```
FAN_POINT(45, 25), \
FAN_POINT(50, 30), \
FAN_POINT(55, 30), \
FAN_POINT(60, 40), \
FAN_POINT(65, 50), \
FAN_POINT(70, 60), \
FAN_POINT(75, 70), \
FAN_POINT(80, 80), \
```

should be the same as

```
FAN_POINT(45, 25), \
FAN_POINT(50, 30), \
FAN_POINT(55, 30), \
FAN_POINT(80, 80), \
```

Reply:
Ok I guess I did understand it better than I thought. So if I only have the line `FAN_POINT(80, 80), \` the first point starts at the 70°C default, but if I add the `FAN_POINT(45, 25), \` it creates a new minimum which is why I'm getting the same results? If so would using just those two points be acceptable, or did I misunderstand and that's what you meant from the beginning?

Reply:
Setting these values overrides the defaults. So with _only_ `FAN_POINT(80, 80)`, the fan comes on at 80°C at 80% and stays at 80% until temps go below 80°C again. (There's the ramp-up/cool-down timings as well, but I'm ignoring those for now.)

Reply:
My reasoning for having the fan turn on at a lower temperature is due to the fact that it was idling around the mid 60's. I would not normally worry about a temperature this low, but to me it was getting hot to the touch on the upper left corner near the escape key and radiating out. This is on a small open stand that sets it at and angle of about 20-25 degrees. It runs hotter laying flat on a surface. This creates a noticeable difference in the temperature between the left and right side of the keyboard after a couple hours of use. Setting it to a lower minimum eliminates this, and keeps the idle temperature around the mid 40's. Is the concern with this that it will cause a shorter life span for the fans?

Reply:
Per your recommendation, I've been testing all day with only the single `FAN_POINT(80, 80)`. I have seen no change in the default behavior. The system still seems to run hot. It's only when adding the other value `FAN_POINT(45, 25)` that I see a temperature drop.

Reply:
That is not my recommendation.

I am saying that what you have is equivalent to:

```
FAN_POINT(45, 25), \
FAN_POINT(50, 30), \
FAN_POINT(55, 30), \
FAN_POINT(80, 80), \
```

Reply:
I'm sorry, that's my fault. I misunderstood. If I'm reading this correctly it should just be those values? If so I'll make that change right away. Did you want a change in the in the `HEATUP` and `COOLDOWN` values as well, or to remove them?

Reply:
Yes, you can just use those points.

Those values should be fine for `HEATUP`/`COOLDOWN`.

Reply:
Thank you for you help and patience on this. I appreciate it. Please let me know if any other changes are necessary.